### PR TITLE
Refactored Edit.jsx.

### PR DESCRIPTION
**Pull request by BhavishyaSahay**
Refactored Edit.jsx from class-based component to functional-based component.

### CODE REVIEW

File: packages/volto/news/6206.feature [hunk 1]
```diff
@@ -0,0 +1 @@
+Refactor the `Edit.jsx` component from a class component to a functional component. @BhavishyaSahay
```
Comment (stevepiercy) [inline packages/volto/news/6206.feature:1]:
```suggestion
Refactor the `manage/Blocks/LeadImage/Edit.jsx` component from a class component to a functional component. @BhavishyaSahay
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]
This may be a duplicate of #4959.

@BhavishyaSahay please read and follow [First-time contributors](https://6.docs.plone.org/contributing/first-time.html), especially Item 2 in [Things not to do](https://6.docs.plone.org/contributing/first-time.html#things-not-to-do).

How is this PR different from #4959?

**Comment by BhavishyaSahay** [comment]
> This may be a duplicate of #4959.
> 
> @BhavishyaSahay please read and follow [First-time contributors](https://6.docs.plone.org/contributing/first-time.html), especially Item 2 in [Things not to do](https://6.docs.plone.org/contributing/first-time.html#things-not-to-do).
> 
> How is this PR different from #4959?

@stevepiercy Sir actually this has been assigned to me as a task for an open source course from my college and I didn't knew that someone has already made the changes before. Please merge it, I'll surely take care of this from the next time and will make sure there is no duplication.

**Comment by stevepiercy** [comment]
@BhavishyaSahay sorry, but we expect all contributors to follow our contributing policy. Please direct your instructor to [Expectations of first-time contributors](https://6.docs.plone.org/contributing/first-time.html#expectations-of-first-time-contributors). I would be happy to discuss with your instructor why this policy is in place.

> If you are a student or learner of Plone, you probably are not a contributor. See the next section, [Expectations of first-time contributors](https://6.docs.plone.org/contributing/first-time.html#expectations-of-first-time-contributors), to determine whether you are in fact a contributor.

Closing as duplicate of #4959 because no distinction was given.

File: pnpm-lock.yaml
Comment (stevepiercy) [inline pnpm-lock.yaml:1]: Remove this file from your pull request.

**Pull request closed without merging**